### PR TITLE
A Beginner's Guide to Swift Contribution Proposal

### PR DESCRIPTION
**Pull request by AndrewLitteken**
A proposal created off of Issue #38

### CODE REVIEW

**Comment by AndrewLitteken** [comment]
I can host and edit the episode if needed.  I have some experience, but I am in no way an expert at it.

**Comment by BasThomas** [comment]
Let me know if I can help out with this one!

**Comment by theblixguy** [comment]
cc @lattner @JohnSundell to check if they would be interested

**Comment by lattner** [comment]
I'm happy to participate as a cohost!

**Comment by AndrewLitteken** [comment]
I'm really excited by everyone that has said that they would be willing to work on this episode!  It seems like (based on current response) a good hosting combination could be:
- @lattner as someone with an understanding of the community and has seen the community grow
- @theblixguy as someone who has become part of the contributing community
- Myself as someone trying to figure out how to get more involved

I also took a look at the first episode edit and it seems pretty much in keeping with how I edit my own podcast, so I would be willing to take that job on.

File: Proposals/XXXX-a-beginners-guide-to-swift-contributions.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# A Beginner's Guide to Contributing to Swift
+
+* Proposal: [PR#22](https://github.com/SwiftCommunityPodcast/podcast/pull/22)
+
+* Authors: [Andrew Litteken](https://github.com//AndrewLitteken)
+
+* Review Manager: Unassigned
```
Comment (tinder-garricnahapetian) [inline Proposals/XXXX-a-beginners-guide-to-swift-contributions.md]: @BasThomas do you want to be the Review Manager on this one? We haven't quite nailed down what that even means but I think its something like: make sure episodes is source compatible, help proposal author push it forward.

File: Proposals/XXXX-a-beginners-guide-to-swift-contributions.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+## Roles
+
+- Producers: Andrew Litteken, Garric Nahapetian
+
+- Lead Host: Open (I am willing to try this, I'm still a beginner podcaster though)
+
+- Co-hosts: Suyash Srijan (By suggestion from Robert Clegg)
+
+- Editor: open
```
Comment (tinder-garricnahapetian) [inline Proposals/XXXX-a-beginners-guide-to-swift-contributions.md]: If @AndrewLitteken can edit then please feel free to take on this role. I can always do it if need be though.

**Comment by tinder-garricnahapetian** [comment]
I'm really excited to see this one moving forward.

**Comment by garricn** [comment]
What do we need to help move this one forward?

**Comment by AndrewLitteken** [comment]
I think w should probably create a bit more of an overview for the episode like they have in the state of the community proposal

**Comment by AndrewLitteken** [comment]
I've created a brief outline of what I think the episode could look like, feel free to make suggestions

**Comment by BasThomas** [comment]
This is looking good — anything we can do to help this forward now there's an overview?

**Comment by theblixguy** [comment]
Looks good to me. We should decide when we want to record this. Perhaps we can do this in the first week of May? What time/day is everyone free? We could do it on the weekend maybe!

**Comment by AndrewLitteken** [comment]
The first week of May should work for me. The weekend (May 4th and 5th) I think should work well for me. Is there a good way for all of us to connect and talk about the specifics?

**Comment by garricn** [comment]
> The first week of May should work for me. The weekend (May 4th and 5th) I think should work well for me. Is there a good way for all of us to connect and talk about the specifics?

@AndrewLitteken I'm glad you asked. For the last episode, @JohnSundell had everyone email him their iMessage contact to handle scheduling.

**Comment by theblixguy** [comment]
Yeah, I think the weekend would be better as I believe everyone will be working Mon-Fri and it could be hard to get a day-off sometimes. 

@lattner What does your schedule look like for May? :)

**Comment by garricn** [comment]
I wonder if we could somehow incorporate the fact that try swift San Jose is happening during wwdc.

**Comment by theblixguy** [comment]
In what way? I guess one way we could connect is if more people tried Swift then we’d have a lot more feedback to work with and provide valuable improvements to the language that benefits everyone.

**Comment by BasThomas** [comment]
Well, the idea of try! Swift San Jose is a hands-on workshop to contribute to Swift, so I think it's doable to include and mention that :)

**Comment by theblixguy** [comment]
Oh! I have never been to one so I didn't know that! Yes, it would be great to mention that as well 😄

**Comment by BasThomas** [comment]
https://www.tryswift.co/events/2019/sanjose/

**Comment by AndrewLitteken** [comment]
Anyone who wants to be involved in the scheduling of this one, let’s do it like they did for the last episode. Email me your iMessage contact at andrew.litteken@gmail.com (or if you’d rather not do that we can keep it to email communication)

**Comment by theblixguy** [comment]
Sent you an email @AndrewLitteken!

**Comment by lattner** [comment]
Sorry for being MIA lately, but I'm also emailing you @AndrewLitteken!

**Comment by AndrewLitteken** [comment]
Quick Update: everyone is in contact with one another and we are planning to record the episode Saturday afternoon (West Coast time)!

**Comment by BasThomas** [comment]
Exciting!

**Comment by AndrewLitteken** [comment]
Also, just for documenting purposes, we'll be using Trycast.  I think it'll support the hosts plus two people, so when we record I'll put the link in this PR for anyone who wants to listen in as Suyash, Chris and I record on Saturday.

**Comment by AndrewLitteken** [comment]
Episode recorded, @JohnSundell can you email me/Twitter DM for a few pointers on the process you use to edit the episode?

**Comment by lattner** [comment]
This lgtm, lemme know when you want to merge it

**Comment by BasThomas** [comment]
Excited for this one 🤩

**Comment by AndrewLitteken** [comment]
The [show document](https://docs.google.com/document/d/1-0FxbuYLaZwaUPxzN8FI2woEq9nEd_sSHa_py9OnoQ8/edit?usp=sharing) for this episode so those interested for future episodes can use it as a starting point.

**Comment by theblixguy** [comment]
I think we can merge this now as the podcast has now been published 🙂

**Comment by BasThomas** [comment]
https://twitter.com/basthomas/status/1134181539191894016 🎉

**Pull request merged**